docs/testplan/snappi-tests/capacity_test.md

```diff
@@ -0,0 +1,70 @@
+# SONiC Switch Capacity Test
+
+## Test Objective
+
+This test aims to assess the true capacities of SONiC switches.
+
+## Test Setup
+
+The test is designed to be topology-agnostic, meaning it does not assume or impose a specific setup wiring, settings, or configuration. The running script will track power usage, temperature, queue watermark, and health state of all SONiC switches in a testbed.
+
+## Test Steps
+
+1. Run all traffic generators at full line rate, injecting traffic into SONiC devices to maximize stress. The test duration is configurable, with a default setting of 1 hour.
+2. Using the switch commands or RPC (Remote Procedure Call), collect all SONiC switches’ metrics listed in the previous section. The data sampling rate is also configurable with a default value of every 1 minute.
+3. Save the sampled raw data to a database via the telemetry interface provided by the SONiC team. The metrics are stored as data points in our database. An example of how to use the interface is provided in telemetry folder.
+
+     ![overview](./capacity_test_diagram.png)
+
+## Metrics are organized with two sets of labels in the database
+
+- **Common labels**: These labels are shared across all metrics within one test job and must be included with every metric.
+
+     ```python
+     METRIC_LABEL_TESTBED: Final[str] = "test.testbed"
+     METRIC_LABEL_TEST_BUILD: Final[str] = "test.os.version"
+     METRIC_LABEL_TEST_CASE: Final[str] = "test.testcase"
+     METRIC_LABEL_TEST_FILE: Final[str] = "test.file"
+     METRIC_LABEL_TEST_JOBID: Final[str] = "test.job.id"
+     ```
+
+- **Metric labels**: These labels identify the specific device and component from which a metric is collected.
```

Comment:
list all metrics below with 1 metric as a separate section

\### PSU metrics

...

\### Sensor metrics

...

and so on